packages/KCard/KCard.vue

```diff
@@ -95,6 +95,7 @@ export default {
     display: flex;
     align-items: center;
     margin-bottom: 1rem;
+    min-height: 38px;
```

Comment:
```suggestion
    min-height: 38px;
    
    .k-button {
      min-height: 38px;
    }
```

We can fix the two pixel diff between buttons with icon only vs buttons with text only by also setting the min-height on the button.